src/api/.keystone/config.js

```diff
@@ -33,9 +33,33 @@ __export(keystone_exports, {
   default: () => keystone_default
 });
 module.exports = __toCommonJS(keystone_exports);
-var import_config2 = require("dotenv/config");
+var import_config3 = require("dotenv/config");
 var import_core2 = require("@keystone-6/core");
 
+// middleware/headers/check-api-key/index.ts
+var import_config = require("dotenv/config");
+var { API_KEY } = process.env;
+var checkApiKey = (req, res, next) => {
+  const { "x-api-key": xApiKey } = req.headers;
+  if (xApiKey !== API_KEY) {
+    return res.status(401).send("Unauthorised");
+  }
+  next();
+};
+var check_api_key_default = checkApiKey;
```

Comment:
what's the point of this 'default' var? Is there going to be other overridden functions?